docs/dev/fundamentals/reading-games-code.md

```diff
@@ -0,0 +1,97 @@
+---
+description: This article teaches how to use ILSpy in order to read game's code
+---
+
+# Reading game's code
+## Introduction
+Chances are, early on in your modding journey you will be met by quite a conundrum - You want to know how something in the game works, but there is an issue! You cannot just open the game's files to find it *(you silly goose)*. 
+
+You see, the way Unity games work, is that their code gets compiled into a .dll file, which you cannot normally open. 
+
+Fortunately, there is something called a "decompiler". What it does, is it grabs game's code and reverses the process, resulting in readable code. While it is not perfect, it gets it's job done.
+
+::: warning
+Please keep in mind that while decompiling and reading the code is *not* illegal, distributing said decompiled code *is* illegal. Same applies for sharing the game's files. 
+
+What this means is that you **CANNOT** bundle any of the game's files with your mods.
+:::
+
+## ILSpy
+[ILSpy](https://github.com/icsharpcode/ILSpy/) is an open-source .NET decompiler.
+
+Downloading it is pretty simple, go to the [Releases](https://github.com/icsharpcode/ILSpy/releases/latest) page and download an installer that matches you system. 
+
+::: tip
+Apart from ILSpy, there is another popular decompiler called "dnSpyEx". Picking which one to use is up to personal preference, but for this article ILSpy will be used due to it's simplicity.
+:::
+
+## Opening the game DLL
+Upon starting ILSpy you will be met with a window similar to this.
+![Screenshot of empty ILSpy window](/images/reading-games-code/ilspy-window.png)
+
+From then you want to open the `Assembly-CSharp.dll` file present in the game's files. 
+
+To get it you need to do the following:
+1. Open game files. On Steam it's done as simple as right clicking the game and pressing 'Browse local files'.
```

Comment:
Consider making these points h3 headings to make them more see-able and appear in the TOC on the right.